src/theme/src/components/input.js

```diff
@@ -0,0 +1,85 @@
+import { minorScale } from '../../../scales'
+
+export default function getInputStyles(theme) {
+  const { tokens } = theme
+
+  return {
+    baseStyle: {
```

Comment:
Should this also include `outline: 'none'` and some similar styles that buttons have?